log/format.go

```diff
@@ -105,6 +105,8 @@ func TerminalFormat(usecolor bool) Format {
 
 		b := &bytes.Buffer{}
 		lvl := r.Lvl.AlignedString()
+		unixTime := float64(r.Time.UnixNano())/1000000
```

Comment:
why is this `1,000,000` instead of `1,000,000,000` for nanoseconds --> seconds?

Reply:
lol Because I'm dumb. will fix this.